package.json

```diff
@@ -79,6 +79,7 @@
     "react-dom": "^17.0.2",
     "react-ga": "^3.1.2",
     "react-gtm-module": "^2.0.11",
+    "react-icons": "^4.8.0",
```

Comment:
Are we using these icons anywhere in the PR? I don't see any imports for this library.